ui/src/views/compute/wizard/ComputeOfferingSelection.vue

```diff
@@ -157,11 +157,11 @@ export default {
             (item.iscustomized === true && maxCpuNumber < this.minimumCpunumber))) {
           disabled = true
         }
-        if (disabled === false && this.minimumCpuspeed > 0 && maxCpuSpeed && maxCpuSpeed !== this.minimumCpuspeed) {
+        if (disabled === false && this.minimumCpuspeed > 0 && maxCpuSpeed && maxCpuSpeed < this.minimumCpuspeed) {
           disabled = true
         }
         if (disabled === false && maxMemory && this.minimumMemory > 0 &&
-          ((item.iscustomized === false && maxMemory !== this.minimumMemory) ||
+          ((item.iscustomized === false && maxMemory < this.minimumMemory) ||
             (item.iscustomized === true && maxMemory < this.minimumMemory))) {
```

Comment:
so,
```suggestion
          (maxMemory < this.minimumMemory) //  && (item.iscustomized === false || item.iscustomised == true)
```
?

Reply:
> so,
> 
> ?

+1, `maxMemory < this.minimumMemory` should be fine. @nvazquez can you check and update.